lib/coll_8_api/operation.rb

```diff
@@ -59,7 +59,7 @@ def payload
     end
 
     def token
-      options[:token]
+      config.token
```

Comment:
What is your argument for retrieving the token from static configuration? The token will vary, and updating it means updating the global state of the client. It feels risky to me.

Reply:
I decided to change it when I was recording the cassettes (I suppose I could simply fetch the ENV var add pass it along with the payload as originally envisaged. 

I just thought that as the main app will be caching it (they expire after 24 hours), it could store in the config. 

Anyway, I can change that back.

Reply:
Does the API accept the same token for each endpoint? If it does, then I think there's no reason to change back.

Reply:
No, I think you are right and it should be changed back. It will be stored in the cache by the main app so can't exactly be loaded in the initializer. I will revert this change and put it back in the options hash.